src/integration-test/java/com/scalar/db/transaction/consensuscommit/ConsensusCommitIntegrationTestBase.java

```diff
@@ -841,14 +837,14 @@ public void get_GetGivenForDeletedWhenCoordinatorStateNotExistAndExpired_ShouldA
   public void getAndScan_CommitHappenedInBetween_ShouldReadRepeatably()
       throws CrudException, CommitException, UnknownTransactionStatusException {
     // Arrange
-    DistributedTransaction transaction = manager.start();
+    ConsensusCommit transaction = manager.start();
```

Comment:
Just a question. Is there any reason of using  `ConsensusCommit` instead of `DistributedTransaction`?
(usually we should use `DistributedTransaction`, right?)

Reply:
Thank you for reviewing this!

The reason of those changes is that the integration tests are actually just for `ConsensusCommit`, but not for general `DistributedTransaction` (I found that when I tried to use it for gRPC transaction but I couldn't). To highlight that, I made those changes, but they are not very important. And yes, usually we should use DistributedTransaction.